packages/attribute-registry-process/package.json

```diff
@@ -41,6 +41,7 @@
     "connection-string": "^4.3.6",
     "dotenv-flow": "^3.2.0",
     "express": "^4.18.2",
+    "mongodb": "5.6.0",
```

Comment:
this should be already included in the commons helper used
```suggestion
```

Reply:
For now it's used in [this line](https://github.com/pagopa/interop-be-monorepo/pull/92/files#diff-cda50d168ca547dfedbc0861e72842888a70aac3daedeecb1e4a87f3f1f6c276R1), when we will refactor the helpers as we discussed it can be removed